src/TAction.cpp

```diff
@@ -197,17 +197,6 @@ void TAction::expandToolbar(TToolBar* pT)
         // This applies the CSS for THIS TAction to a CHILD's representation on the Toolbar
         button->setStyleSheet(css);
 
-        /*
```

Comment:
The questing is still relevant isn't it -even though it never got answered...
:man_shrugging:

Reply:
I'm not sure, it's a message more than a decade old. If it's relevant it'll come back one way or the other.